examples/Example5/CMakeLists.txt

```diff
@@ -1,14 +1,10 @@
 # SPDX-FileCopyrightText: 2020 CERN
 # SPDX-License-Identifier: Apache-2.0
 
-if(NOT TARGET g4HepEm)
+if(NOT TARGET G4HepEm::g4HepEm)
```

Comment:
Is this needed? I think we should use the target consistently, either with or without scope (see below and `example6`).

Reply:
Good point on the scope, I've changed the remainder of G4HepEm consumers to use the scoped target names.